### ISSUE TITLE
Difficulty signing into Oppia website.

### ISSUE THREAD

**Issue by Norrno** [oppia/lessons-team#1099]
For over 24 hours now, I have encountered  difficulty submitting and logging in.

**Comment by Ayomiiide** [comment]
Hello @Norrno I had this same issue, but when i selected the option that says 'do not send these emails' it worked and signed me in

**Comment by Norrno** [comment]
Alright,  let me try again.
Thank you so much.

**Comment by Norrno** [comment]
> Hello @Norrno I had this same issue, but when i selected the option that says 'do not send these emails' it worked and signed me in

it worked, thank you soooo much

**Comment by Ayomiiide** [comment]
You are welcome

**Comment by Smart3990** [comment]
@Ayomide after selecting it you can change that preference at the profile side.

**Comment by seanlip** [comment]
This is a known bug, thanks for reporting. Closing as a duplicate of https://github.com/oppia/oppia/issues/19858
